### PR TITLE
Refs [TASK] [34651] [Create UI Setting Screen]

### PR DESCRIPTION
**Pull request by huydd-2678**
## Related Tickets
- [#Task 34651: Create UI Setting Screen](https://edu-redmine.sun-asterisk.vn/issues/34651)
## WHAT
Detail UI: https://marvelapp.com/prototype/6j24jfh/screen/77555847
- Create Textviews to show titles
- Create Spinner to choose language
- Create Switch to turn on/off notification
## Evidence (Screenshot or Video)
Screenshot_2021-03-29-07-35-11-867_com li weatherapp
Screenshot_2021-03-29-13-19-29-059_com li weatherapp
Screenshot_2021-03-29-13-19-34-875_com li weatherapp

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/li/weatherapp/ui/setting/SettingFragment.kt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import com.li.weatherapp.R
+import com.li.weatherapp.base.BaseFragment
+import com.li.weatherapp.utils.removeFragment
+import kotlinx.android.synthetic.main.fragment_setting.*
+
+class SettingFragment : BaseFragment() {
+
+    override val layoutResource: Int = R.layout.fragment_setting
```
Comment (huytq-2422) [inline app/src/main/java/com/li/weatherapp/ui/setting/SettingFragment.kt]:
```suggestion
    override val layoutResource: Int  get() = R.layout.fragment_setting
```

Comment (huydd-2678) [reply app/src/main/java/com/li/weatherapp/ui/setting/SettingFragment.kt]: em fix rồi ạ

File: app/src/main/java/com/li/weatherapp/ui/setting/SettingFragment.kt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                ) {
+                }
+
+                override fun onNothingSelected(parent: AdapterView<*>) {
+                }
+            }
+        }
+    }
+
+    private fun onTextAboutClick() {
```
Comment (huytq-2422) [inline app/src/main/java/com/li/weatherapp/ui/setting/SettingFragment.kt]: Đặt tên theo chức năng của hàm 
```suggestion
    private fun showNotificationDialogABCXYZ() {
```

File: app/src/main/java/com/li/weatherapp/ui/setting/SettingFragment.kt [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            val builder = AlertDialog.Builder(context, R.style.AboutDialogTheme)
+            builder.apply {
+                setTitle(resources.getString(R.string.title_about))
+                setMessage(resources.getString(R.string.text_about_message))
+                setNegativeButton(resources.getString(R.string.text_ok)) { dialog, _ ->
+                    dialog.cancel()
+                }
+                val alertDialog: AlertDialog = builder.create()
+                alertDialog.show()
+            }
```
Comment (huytq-2422) [inline app/src/main/java/com/li/weatherapp/ui/setting/SettingFragment.kt]: không viết liền đoạn này được sao ? 
```suggestion
            AlertDialog.Builder(context, R.style.AboutDialogTheme)
                .setTitle(resources.getString(R.string.title_about))
                .setMessage(resources.getString(R.string.text_about_message))
                .setNegativeButton(resources.getString(R.string.text_ok)) { dialog, _ ->
                    dialog.cancel()
                }
                .create()
                .show()
            }
```

**Pull request merged**